mgmt/RecordsConfig.cc

```diff
@@ -518,6 +518,8 @@ static const RecordElement RecordsConfig[] =
   ,
   {RECT_CONFIG, "proxy.config.http.parent_proxy.connect_attempts_timeout", RECD_INT, "30", RECU_DYNAMIC, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
   ,
+  {RECT_CONFIG, "proxy.config.http.update_hostdb_on_parent_proxy_connect_errors", RECD_INT, "0", RECU_DYNAMIC, RR_NULL, RECC_INT, "[0-1]", RECA_NULL}
```

Comment:
yeah it is long, i prefer your second suggestion since it's used in the http transaction to update or not hostdb.  All the ones named spaced to proxy.config.hostdb are internal to hostdb and its configuration.  But if you prefer #1, I'll use it.